package/components/hint/_hint.scss

```diff
@@ -12,4 +12,18 @@
 
     color: $govuk-secondary-text-colour;
   }
+
+  // Reduces margin-bottom of hint when used after the default label (no class)
+  // or govuk-label--s for better vertical alignment.
+
+  // This adjustment will not work when the label is inside the <h1>, however
+  // it is unlikely that the default or govuk-label--s class would be used in
+  // this case.
+
+  // This adjustment will not work in browsers that do not support :not()
+  // Users with these browsers will see the default size margin (5px larger)
+
+  .govuk-label:not(.govuk-label--m):not(.govuk-label--l):not(.govuk-label--xl) + .govuk-hint {
```

Comment:
This change hasn't been documented – can you work with @dashouse to add something to the changelog before we ship?

Reply:
added